wiki/Tournaments/o!cCC/1/en.md

```diff
@@ -0,0 +1,352 @@
+
+| Sunday, 2019-09-29 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| KWYJIBO ![][flag_AU] | 0 | **4** | ![][flag_TH] **Blackbear** | [#1](https://osu.ppy.sh/community/matches/55194369) |
+| **Saaya** ![][flag_VN] | **4** | 0 | ![][flag_RU] MeeeDBeD | -win by default- |
+| mothernature ![][flag_VN] | 2 | **4** | ![][flag_AU] **chickennando** | [#1](https://osu.ppy.sh/community/matches/55198877) |
+| -MomoX ![][flag_PH] | 0 | **4** | ![][flag_BR] **Hanik** | [#1](https://osu.ppy.sh/community/matches/55201762) |
+| DataPata ![][flag_SE] | 3 | **4** | ![][flag_PE] **-Boltic-** | [#1](https://osu.ppy.sh/community/matches/55208924) |
+| Aryssiel ![][flag_CL] | 0 | **4** | ![][flag_CA] **wwwww-** | -win by default- |
+| **Julo** ![][flag_FR] | **4** | 2 | ![][flag_US] wonjae | [#1](https://osu.ppy.sh/community/matches/55208715) |
+| **SchnitzelBae** ![][flag_DK] | **4** | 0 | ![][flag_US] Kisetsunotori | -win by default- |
+
+---
+
+## Ruleset
+
+### General rules
+
+1. The score system used in this tournament will be **Score V2**.
+2. Players will be seeded according to their average rank following the Qualifier Stage.
+3. Anyone participating in this tournament (including staff) must have common decency, and not be overly toxic for no reason, otherwise a ban from the tournament will be enforced.
+4. Only in the first 20 seconds of a map can the map be retried if anything hindering happens such as a disconnect. If anything happens after this, whether it is a disconnect or lags, the map will not be replayed.
+5. If the beatmap ends in a draw, the beatmap has to be replayed again. If it ends in a draw on the second try, the beatmap will be nullified, and the player who picked it will have to choose another map.
+6. Failing a beatmap **still** makes your score count.
+7. Players will be allowed to ban **two maps** per match.
+8. You are only allowed to **ban one map per mod per player**, meaning you can only ban e.g. 1 Hidden map but your opponent can still ban another Hidden map.
+9. The player that rolls highest will pick first and ban second.
+10. Ban alternation is used (Lower rolling player bans one map first, then the higher rolling player bans one map, then the lower rolling, then the higher rolling).
+11. Always remember that this is just a game. (e.g. behave yourself while the match is ongoing)
+12. If neither player shows up to a match, the winner will be decided by who has the best seeding.
+13. Any player no-showing Qualifiers without any explanation will get a 1-year ban from any Kasumii tournaments. Any player who no-shows twice in Bracket stage without any explanation will get a 2-years ban from any of Kasumii tournaments.
+    - Players currently on all running  blacklist can accesed via [this link](https://docs.google.com/spreadsheets/d/e/2PACX-1vR_ZTkA7f3Z_ApxUPtHKkdXncl_RVLA3pwFFsx-Me1RmJ4QpqsoUii-veVw00XJDYtV1XfJEgy7wfw0/pubhtml). Any player potentially banned from my tournaments due to this will be put there./
```

Comment:
where did this slash at end come from hmmmmm

Reply:
That'd be my very light mechanical keyboard's typo